app/services/bulk_upload/lettings/row_parser.rb

```diff
@@ -171,6 +172,16 @@ def log
 
 private
 
+  def validate_lettings_type_matches_bulk_upload
+    if [1, 3, 5, 7, 9, 11].include?(field_1) && bulk_upload.supported_housing?
```

Comment:
Is there no field for needs type specifically?

Reply:
Also, would it make more sense to check that those aren't supported housing options as opposed to checking that they are general needs?

Reply:
needs type is selected as part of the user journey when they select the file to upload. therefore we check to make sure the data from the csv matches the value they selected from the user journey. the needs type for the csv is derived from field 1 which is the `lettype`

Reply:
> Also, would it make more sense to check that those aren't supported housing options as opposed to checking that they are general needs?

sure i can do that. don't think it will make any real difference though

Reply:
I've now inverted the logic